mayastor/src/grpc/mayastor_grpc.rs

```diff
@@ -126,6 +126,7 @@ impl mayastor_server::Mayastor for MayastorSvc {
             let args = request.into_inner();
             let uuid = args.uuid.clone();
             let name = uuid_to_name(&args.uuid)?;
+            info!("Creating nexus '{}' with arguments: '{:?}'", uuid, args);
```

Comment:
Would it be better to use the name rather than the uuid? The uuid is already in the args whereas the name is constructed from the uuid.

Reply:
@tiagolobocastro what about we try:

```
#[instrument(err)]
```
We should see the call with arguments -- all the time (and during debug) so we don't need to add `info!()` calls at all?

Reply:
But that would be only in case of error?

Reply:
Yeah that works, I've removed the log and added that instrument with info.
Though tbh I tend not to like to inspect those instrument thingies, too confusing to visually inspect without any tooling but it's all we have atm..

Reply:
Since we are being honest -- I don't like it either not one bit! The only thing I like is that can add "log in and out". 

The reason for this is its very easy to have args change or returns for that matter and the logging messages are forgotten or not updated. 

OTOH, all things considered it's (to me) one of those "o will look into that later"

I see you have mastered proc-macro powers so we can make look into it later unless it bugs you so much you want to kill it with fire right now :)

Reply:
I actually had a look at this before, we can configure what the instrument macro does, I think mayastor has it "configured for tracing" with all the events showing. On the control plane services, you only get the one log file with the function arguments for example.

I also got the instrument enter/exit thingies to actually show up on the jaeger container at some point before Christmas, might raise a story for that.